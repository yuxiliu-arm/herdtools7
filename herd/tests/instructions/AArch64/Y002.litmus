AArch64 Y002
(* MUL *)

{
    0: X1 = 2;
    0: X2 = 3;
}

P0 ;
MUL X3, X1, X2;

forall (
    0: X1 = 2 /\
    0: X2 = 3 /\
    0: X3 = 6
)
