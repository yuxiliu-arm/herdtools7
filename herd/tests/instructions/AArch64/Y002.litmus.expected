Test Y002 Required
States 1
0:X1=2; 0:X2=3; 0:X3=6;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X1=2 /\ 0:X2=3 /\ 0:X3=6)
Observation Y002 Always 1 0
Hash=83c32d5aaeef22f6d94225d5e348d323

